webui/react/src/pages/TrialDetails.tsx

```diff
@@ -200,6 +202,14 @@ const TrialDetailsComp: React.FC = () => {
         key: TabType.Metrics,
         label: 'Metrics',
       });
+      const hyperparameterTabPosition = tabs.findIndex((item) => {
+        return item.key === TabType.Hyperparameters;
+      });
+      tabs.splice(hyperparameterTabPosition + 1, 0, {
+        children: <TrialCheckpoints experiment={experiment} pageRef={pageRef} trial={trial} />,
+        key: TabType.Checkpoints,
+        label: 'Checkpoints',
+      });
```

Comment:
let's make sure this only shows up when the feature flag is on:
```suggestion
      if (f_flat_runs) {
        const hyperparameterTabPosition = tabs.findIndex((item) => {
          return item.key === TabType.Hyperparameters;
        });
        tabs.splice(hyperparameterTabPosition + 1, 0, {
          children: <TrialCheckpoints experiment={experiment} pageRef={pageRef} trial={trial} />,
          key: TabType.Checkpoints,
          label: 'Checkpoints',
        });
      }
```